presto-cli/pom.xml

```diff
@@ -170,6 +183,96 @@
                     </execution>
                 </executions>
             </plugin>
+
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>rpm-maven-plugin</artifactId>
+                <version>2.2.0</version>
+                <executions>
+                    <execution>
+                        <id>generate-rpm</id>
+                        <goals>
+                            <goal>rpm</goal>
+                        </goals>
+                    </execution>
+                </executions>
+                <configuration>
+                    <name>${project.name}</name>
+                    <version>${project.version}</version>
+                    <release>1</release>
+                    <group>Applications/Databases</group>
+                    <description>Presto CLI RPM Package.</description>
+                    <needarch>x86_64</needarch>
+                    <copyTo>${project.build.directory}/rpm/${project.build.finalName}.x86_64.rpm</copyTo>
+
+                    <mappings>
+                        <mapping>
+                            <directory>/usr/local/bin</directory>
+                            <filemode>755</filemode>
+                            <sources>
+                                <source>
+                                    <location>target/presto-cli-${project.version}-executable.jar</location>
+                                    <destination>presto-cli</destination>
+                                </source>
+                            </sources>
+                        </mapping>
+                    </mappings>
+                </configuration>
+            </plugin>
+
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-enforcer-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>enforce-file-size</id>
+                        <phase>verify</phase>
+                        <goals>
+                            <goal>enforce</goal>
+                        </goals>
+                        <configuration>
+                            <rules>
+                                <requireFilesSize>
+                                    <!-- Maven Central has a 1GB limit -->
+                                    <maxsize>1106000000</maxsize>
```

Comment:
I added this to be consistent with the presto-server-rpm module but I think this is not strictly necessary here since the presto-cli artifacts are much less than 1GB.

Should I remove this in interest of simplicity?

Reply:
`presto-cli-335-SNAPSHOT-executable.jar` is ~15M, so RPM would be about the same size.

Actually, i think it would be a failure if the CLI goes into 3-digits MB. It should remain "small".
I would keep the check, but change the cap to say 25M